README.md

```diff
@@ -28,9 +28,16 @@ The ATAC-seq pipeline specification is also the official pipeline specification
 
 3) Run a pipeline with Caper.
 
+## DNAnexus
+
+You can also run our pipeline on DNAnexus without using Caper or Cromwell. There are two ways to build a workflow on DNAnexus based on our WDL.
+
+1) [dxWDL CLI](docs/tutorial_dx_cli.md)
+2) [DNAnexus Web UI](docs/tutorial_dx_web.md)
+
 ## Conda
 
-We don't recommend Conda for dependency helper. Use Docker or Singularity instead. We will not take any issues about Conda. You can install Singularity locally without super-user privilege and use it for our pipeline with Caper (with `--use-singularity`).
+We don't recommend Conda for resolving dependencies. Use Docker or Singularity instead. We will not take any issues about Conda. You can install Singularity and use it for our pipeline with Caper (by adding `--use-singularity` to command line arguments).
```

Comment:
Jin I think I'd rather say "We no longer recommend Conda for resolving dependencies and plan to phase out Conda support.  Instead we recommend using Docker or Singularity.  You can install Singularity and use it for our pipeline with Caper (by adding `--use-singularity` to command line arguments)."

Reply:
Fixed in commit f532389